src/static_handler.cc

```diff
@@ -74,21 +82,21 @@ RequestHandler::Status StaticHandler::HandleRequest(const Request& req, Response
 
             resp->SetStatus(Response::code_200_OK);
 
-            if (mt ==  MIMEType_md) {
+            if (convert_markdown_) {
```

Comment:
I think we probably want this to be `if (mt == MIMEType_md && convert_markdown_)`. Otherwise I expect that every file will be try to be converted! We only want to both trying if this file being served is a .md file